src/App.jsx

```diff
@@ -0,0 +1,19 @@
+import "./App.scss";
+import Header from "./components/Header/Header";
```

Comment:
Suggestion : Have a common index file for all component and import from there. Something like this,
```js
import {Header,Footer,Home} from "./components/index"
```

Reply:
I'm changing for folder structure PR request. thanks 👍